MDANSE_GUI/Src/MDANSE_GUI/InputWidgets/AtomSelectionWidget.py

```diff
@@ -457,6 +575,37 @@ def helper_dialog(self) -> None:
                 self.helper.restoreGeometry(self.helper.previous_geometry)
             self.helper.show()
 
+    @Slot()
+    def load_selection_from_file_dialog(self) -> None:
+        """Load a selection from a file.
+
+        At the moment it is possible to use .mda files,
+        or JSON text files.
+        """
+        fname = QFileDialog.getOpenFileName(
+            self._base,
+            "Load selection from a file (JSON or MDA)",
+            str(self._trajectory_path),
+            "MDANSE selection files (*.mda *.json);;HDF5 files (*.h5);;HDF5 files(*.hdf);;All files(*.*)",
+        )[0]
+        if fname is None:
+            return
+        if len(fname) < 1:
+            return
+        temp_selection = ReusableSelection()
+        try:
+            temp_selection.load_from_hdf5(fname)
+        except OSError:
+            LOG.warning("File %s could not be read as an HDF5 file", fname)
```

Comment:
Is this a `warning` or just an `info`? I think the only `warning` should be the failure to load.